workload/master_config.yaml

```diff
@@ -0,0 +1,43 @@
+s3api:
+  # TEST_ID: TEST-35748
+  object_size:
+    start: 0Kib
+    end: 100Kib
+  part_range: #Multipart Specific
+    start: 50
+    end: 100
+  min_runtime_to_pass: 2h
```

Comment:
Are we planning to use this across all workload.

Reply:
No, Wherever needed as this is multipart specific.

Reply:
min_runtime_to_pass this change has to be made all over the config files.

Reply:
Yes, we will do once start implementing/considering master config in next phase